api/AudioScheduledSourceNode.json

```diff
@@ -0,0 +1,381 @@
+{
+  "api": {
+    "AudioScheduledSourceNode": {
+      "onended": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioScheduledSourceNode/onended",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": "35",
+                "version_removed": "56",
+                "notes": "Interface implemented as <code>AudioSourceNode</code>."
+              },
+              {
+                "version_added": "14",
+                "version_removed": "34",
+                "prefix": "-webkit-"
+              }
+            ],
+            "edge": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "25",
+                "version_removed": "52",
+                "notes": "Interface implemented as AudioSourceNode."
```

Comment:
This should be on the interface compat status, and using the "alternate_name" methodology